src/envoy/http/mixer/report_data.h

```diff
@@ -25,9 +25,8 @@
 namespace Envoy {
 namespace Http {
 namespace Mixer {
-static const std::string permissive_policy_id_field =
-    "shadow_effective_policyID";
-static const std::string permissive_resp_code_field = "shadow_response_code";
+static const std::string permissivePolicyIDField = "shadow_effective_policyID";
+static const std::string permissiveRespCodeField = "shadow_response_code";
```

Comment:
const name always started with k.  So it should be kRabcPolicyIDField

Reply:
renamed(keep permissive)